src/query/FluxParser.cpp

```diff
@@ -201,7 +201,7 @@ bool FluxQueryResult::next() {
 }
 
 FluxDateTime *FluxQueryResult::convertRfc3339(String value, const char *type) {
-    FluxDateTime *ft = nullptr;
+    // unused declaration FluxDateTime *ft = nullptr;
```

Comment:
Why just commenting? Remove this, please.

Reply:
OK, have removed the line and comitted back in on the branch